docs/BREAKINGCHANGES.md

```diff
@@ -0,0 +1,217 @@
+Previously, the output while using `Get-Content -Delimiter` was inconsistent and inconvient as it required further processing of the data to remove the delimiter.
+This change removes the delimiter in returned lines.
+
+
+### Implement Format-Hex in C# [#3320](https://github.com/PowerShell/PowerShell/issues/3320)
+ The `-raw` parameter is now No-op. Going forward all of the output will be displayed with a true representation of numbers that includes all of the bytes for its type (what the Raw parameter was formally doing).
+
+
+### PowerShell as a default shell doesn't work with script command [#3319](https://github.com/PowerShell/PowerShell/issues/3319)
+On Unix, it is a convention for shells to accept `-i` for an interactive shell and many tools expect this behavior (`script` for example, and when setting powershell as the default shell) and calls the shell with the `-i` switch. This change is breaking in that -i previously could be used as short hand to match `-inputformat` which now will need to be `-in`.
+
+
+### Completions for environment variables differ between platforms [#3227](https://github.com/PowerShell/PowerShell/issues/3227)
+The casing of `$PSModulePath` has been corrected to be consistent with Windows PowerShell.
+TODO: not sure if this actually a breaking change.  If the change is to make something consistent with Windows, there would be no difference between 6.0 and 5.1
+
+
+### Typo fix in Get-ComputerInfo property name [#3167](https://github.com/PowerShell/PowerShell/issues/3167)
+`BiosSerialNumber` was misspelled as "BiosSeralNumber".
+
+
+### Add Get-StringHash and Get-FileHash cmdlets [#3024](https://github.com/PowerShell/PowerShell/issues/3024)
+TODO: Labeled with breaking change so the difference between windows and core can be described
+
+
+### Add validation on get-* cmdlets where passing $null returns all objects instead of error [#2672](https://github.com/PowerShell/PowerShell/issues/2672)
+Passing `$null` to any of the following will now throw an error: `Get-Credential -UserName`, `Get-Event -SourceIdentifier`, `Get-EventSubscriber -SourceIdentifier`, `Get-Help -Name`, `Get-PSBreakpoint -Script`, `Get-PSProvider -PSProvider`, `Get-PSSessionConfiguration -Name`, `Get-PSSnapin -Name`, `Get-Runspace -Name`, `Get-RunspaceDebug -RunspaceName`, `Get-Service -Name`, `Get-TraceSource -Name`, `Get-Variable -Name`, `Get-WmiObject -Class`, and `Get-WmiObject -Property`.
+
+
+### Add support W3C Extended Log File Format in Import-Csv [#2482](https://github.com/PowerShell/PowerShell/issues/2482)
+Previously, the `Iport-Csv` cmdlet cannot be used to directly import the log files in W3C extended log format and additional action would be required.
+With this change, W3C extended log format is supported.
+
+
+### Parameter binding problem with ValueFromRemainingArguments in PS functions [#2035](https://github.com/PowerShell/PowerShell/issues/2035)
+TODO
```

Comment:
ValueFromRemainingArguments now returns the values as an array instead of a single value which itself is an array